Dockerfile

```diff
@@ -1,15 +1,4 @@
-FROM ubuntu:18.04
-RUN apt-get update -y && apt-get install -y \
-python3-pip \
-python3-setuptools \
-libsystemd-dev \
-# Required for Postgres
-libpq-dev \
-python3-dev
-
-WORKDIR /code
-COPY . .
-RUN pip3 install -r requirements.txt
+FROM quay.io/redhat-beyond/beyond
 EXPOSE 5000
 
 ENTRYPOINT python3 app.py runserver > web-application.log
```

Comment:
Where is this Dockerfile used?

Reply:
Dockerfile is used by docker-compose in both of the following cases:

1. In the github flow, it is being invoked from here:

.github/workflows/main.yml
    - name: Build the docker-compose stack
      run: docker-compose up -d

2. In the Vagrant flow (dev, stage, prod) it is being invoked from here:

Vagrantfile
  config.vm.provision :docker_compose, yml: "/vagrant/docker-compose.yml", rebuild: true, run: "always",
   compose_version: "1.28.2"

Reply:
I don't understand the code. There seems to be a circle dependency. docker-compose is building the same images as the parent image.

Reply:
You Are right. I was confused.
There is no need for 'Docker' file anymore. Because the 'beyond' image is now being pulled from quayi.
I am removing the Docker file.

Reply:
What happens if you want to update the image? Is it going to be done manually?

Reply:
well, I assume that this is going to be a different flow for building and pushing the image...
Can you suggest how should I start this flow?

Reply:
Can you please google a bit? Take a look at my other comments please.